### ISSUE TITLE
It looks like some vim' command don't work when enable modal editing(Vim like)

### ISSUE THREAD

**Issue by kmephistoh** [lapce/lapce#104]
when enable modal editing(Vim like), some vim' command don't work, such as:
dd db di yy dw

**Comment by jayy-lmao** [comment]
Yeah still working through the essentials. Might help contributors / maintainer to have a priority list like you've done.
I agree that deleting and yanking are must haves for me. 

The other one I really pine for is "gd". Seeing as Lapce already has a "Go to definition" feature it should be possible :)

**Comment by bugadani** [comment]
Thanks for reporting! Could you please see if the commands are still not implemented? There were a few changes made to the key maps recently.

**Comment by kmephistoh** [comment]
> Thanks for reporting! Could you please see if the commands are still not implemented? There were a few changes made to the key maps recently.

I pull the main branch, version v0.0.12, compile and test in Ubuntu 21.10:
**dd db yy dw**  works now,  thanks for your grreat job.
**di** not works, sometime you may want delete a string inside quotes, such as "string",  **di"** can do it

**Comment by dzhou121** [comment]
di still needs to be implemented. We're planning to use the syntax tree parsed by Tree-Sitter to support powerful "insides" just like how helix editor does it.
